grails-app/migrations/core/2018-05-23-fix-product-images.groovy

```diff
@@ -38,7 +38,7 @@ databaseChangeLog = {
 					}
 
 					// scale image to thumbnail
-					byte[] thumb = new ImageResizer().resize(bytes, "thumbimage" + ext, ImageResizer.Size.THUMB)
+					byte[] thumb = new ImageResizer().resize(bytes, "thumbimage" + ext, ImageResizer.Size.PRODUCT_THUMB)
```

Comment:
Hmm changing a migration retroactively might actually produce an error in production, as the migration content gets hashed and checked. I'm not 100% sure about this. If there's any problem, it should come up when this gets deployed to staging, as there will be a database state with the unchanged migration already applied. So safe to ignore for now, but let's keep an eye on staging deployment.

Reply:
Lets see!